### ISSUE TITLE
Lemma me vs I

### ISSUE THREAD

**Issue by amir-zeldes** [UniversalDependencies/UD_English-Pronouns#2]
Hi and thanks for contributing this interesting dataset - I was wondering why the lemma of 'mine' is 'me' and not 'I'. The lemma of 'hers' seems to be 'she', so wouldn't we expect 'I' as the first person lemma?

Another thing to consider is cross-treebank comparability: in other UD English corpora (GUM, EWT) the lemma of the attributive possessive pronouns is the attributive possessive itself, for the substitutive possessive also itself, and for the other forms it's the subject form, so we get:

  * I -> I
  * me -> I
  * my -> my
  * mine -> mine

If there's no special reason to change these, maybe the lemmas could be changed to match the other corpora? See also UniversalDependencies/docs#517

**Comment by AngledLuffa** [comment]
I'm look at forking this dataset.  Are you thinking

```
She wants to drive -> she
Give her the keys -> she
This is her car -> her
It is hers -> hers
```

In fact, this dataset only seems to have the 4th type.  Perhaps it should be expanded to include a few instances of the others as well.

**Comment by amir-zeldes** [comment]
Yes, that's how I would lemmatize those as well.

**Comment by AngledLuffa** [comment]
I think this issue can be closed, although perhaps you want to check my work first @amir-zeldes

**Comment by amir-zeldes** [comment]
Thanks @AngledLuffa - I guess my only comment on this is that on closer inspection, the FEAT `Case=Gen` is kind of strange for the substitutive possessive, since in sentence context it is actually a matrix argument. So in "Mine is nice" I would have said "Mine" is nominative, and the xpos PRP (rather than PRP$) suggests this as well IMO.

**Comment by nschneid** [comment]
Yeah looks like it should be `Poss=Yes`: https://universaldependencies.org/u/feat/Poss.html

**Comment by dan-zeman** [comment]
> ..., the FEAT `Case=Gen` is kind of strange for the substitutive possessive, ...

This should be now fixed (https://github.com/UniversalDependencies/UD_English-Pronouns/pull/5).
